vod/thumb/thumb_grabber.c

```diff
@@ -55,7 +55,9 @@ static codec_id_mapping_t codec_mappings[] = {
 	{ VOD_CODEC_ID_HEVC, AV_CODEC_ID_H265, "h265" },
 	{ VOD_CODEC_ID_VP8, AV_CODEC_ID_VP8, "vp8" },
 	{ VOD_CODEC_ID_VP9, AV_CODEC_ID_VP9, "vp9" },
+#if LIBAVFORMAT_VERSION_INT >= AV_VERSION_INT(60, 16, 100)
```

Comment:
need to check libavcodec version, not libavformat, this module doesn't use libavformat...
I think it should be LIBAVCODEC_VERSION >= AV_VERSION_INT(57, 25, 0)
the codec id constant was added here - https://github.com/FFmpeg/FFmpeg/commit/d42809f9835a4e9e5c7c63210abb09ad0ef19cfb

Reply:
Updated to libavcodec with 57,89,100